scripts/os_configure/install_software_22.bash

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/bash
+#
+# install_software_20.bash [computer_type:{office, robot, default:lab}]
+#
+
+script_own_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
+SCRIPT_PATH=$script_own_dir
+
+computer_type=$1
+if [ -z "$1" ]
+then
+  computer_type="lab"
+  echo "Computer type not provided using default $computer_type"
+fi
+
+echo "Installing software for $computer_type computers. Press <ENTER> to continue."
+read
+
+ROS_VERSION=melodic
+ROS2_VERSION=foxy
+
+# Core tools
+sudo apt update && sudo apt -y install curl gnupg2 lsb-release
+
+#KDE Backports
+sudo apt-add-repository -y ppa:kubuntu-ppa/backports
+sudo apt update && sudo apt -y dist-upgrade && sudo apt -y autoremove
+
+# Nala - better apt frontend
+echo "deb http://deb.volian.org/volian/ scar main" | sudo tee /etc/apt/sources.list.d/volian-archive-scar-unstable.list
+wget -qO - https://deb.volian.org/volian/scar.key | sudo tee /etc/apt/trusted.gpg.d/volian-archive-scar-unstable.gpg > /dev/null
+sudo apt update
```

Comment:
I think we don't need this anymore. This should be in the repositories.

Reply:
Yes but for 20.04 i think we still need it

Reply:
Ok maybe it's not even in the 22.04 packages

Reply:
Would leave this for now as is

Reply:
still needed to add in 22.04